src/NzbDrone.Core/MediaFiles/MediaInfo/VideoFileInfoReader.cs

```diff
@@ -83,85 +83,101 @@ public MediaInfoModel GetMediaInfo(string filename)
+                        int.TryParse(mediaInfo.Get(StreamKind.Audio, AudioIndex, "PlayTime"), out audioRuntime);
 
-                    int.TryParse(aBitRate, out audioBitRate);
-                    int.TryParse(mediaInfo.Get(StreamKind.Audio, 0, "StreamCount"), out streamCount);
+                        string aBitRate = mediaInfo.Get(StreamKind.Audio, AudioIndex, "BitRate").Split(new string[] { " /" }, StringSplitOptions.None)[0].Trim();
 
+                        int.TryParse(aBitRate, out audioBitRate);
 
-                    string audioChannelsStr = mediaInfo.Get(StreamKind.Audio, 0, "Channel(s)").Split(new string[] { " /" }, StringSplitOptions.None)[0].Trim();
+                        string audioChannelsStr = mediaInfo.Get(StreamKind.Audio, AudioIndex, "Channel(s)").Split(new string[] { " /" }, StringSplitOptions.None)[0].Trim();
 
-                    var audioChannelPositions = mediaInfo.Get(StreamKind.Audio, 0, "ChannelPositions/String2");
-                    var audioChannelPositionsText = mediaInfo.Get(StreamKind.Audio, 0, "ChannelPositions");
+                        var audioChannelPositions = mediaInfo.Get(StreamKind.Audio, AudioIndex, "ChannelPositions/String2");
+                        var audioChannelPositionsText = mediaInfo.Get(StreamKind.Audio, AudioIndex, "ChannelPositions");
 
-                    string audioLanguages = mediaInfo.Get(StreamKind.General, 0, "Audio_Language_List");
+                        string audioProfile = mediaInfo.Get(StreamKind.Audio, AudioIndex, "Format_Profile").Split(new string[] { " /" }, StringSplitOptions.None)[0].Trim();
+                        int.TryParse(audioChannelsStr, out audioChannels);
 
-                    string videoProfile = mediaInfo.Get(StreamKind.Video, 0, "Format_Profile").Split(new string[] { " /" }, StringSplitOptions.None)[0].Trim();
-                    string audioProfile = mediaInfo.Get(StreamKind.Audio, 0, "Format_Profile").Split(new string[] { " /" }, StringSplitOptions.None)[0].Trim();
+                        mediaInfoModel.AudioStreams.Add(new AudioInfoModel
+                        {
+                            AudioFormat = mediaInfo.Get(StreamKind.Audio, AudioIndex, "Format"),
+                            AudioCodecID = mediaInfo.Get(StreamKind.Audio, AudioIndex, "CodecID"),
+                            AudioProfile = audioProfile,
+                            AudioCodecLibrary = mediaInfo.Get(StreamKind.Audio, AudioIndex, "Encoded_Library"),
+                            AudioAdditionalFeatures = mediaInfo.Get(StreamKind.Audio, AudioIndex, "Format_AdditionalFeatures"),
+                            AudioBitrate = audioBitRate,
+                            AudioChannels = audioChannels,
+                            AudioChannelPositions = audioChannelPositions,
+                            AudioChannelPositionsText = audioChannelPositionsText,
+                            Language = mediaInfo.Get(StreamKind.Audio, AudioIndex, "Language/String"),
```

Comment:
Should we just pull `Language` instead of `Language/String`? That would give us the iso 2 digit code from the get go and we wouldn't have to convert it back from String to Code in FileNameBuilder.cs.

Reply:
Wouldn't that break things like pt-br since there is no 2 digit code for it ? https://github.com/Radarr/Radarr/pull/3877

Reply:
Based on description of that property, should work fine

2-letter ISO 639-1 if exists, else 3-letter ISO 639-2, and with optional ISO 3166-1 country separated by a dash if available, e.g. en, en-us, zh-cn